sections/migrations.js

```diff
@@ -292,8 +292,8 @@ export default [
   {
     type: "method",
     method: "rollback",
-    example: "knex.migrate.rollback([config])",
-    description: "Rolls back the latest migration group.",
+    example: "knex.migrate.rollback([config], [all])",
+    description: "Rolls back the latest migration group. The `all` parameter can be used to rollback all applied migrations, instead of just the last batch.",
```

Comment:
Supported value of 'all' can be clarified - "if `all parameter is truthy, all applied migrations <...>`".

Reply:
Implemented changes.